File: unittest/scripts/auto/js_shell/scripts/mysql_classic_result_help_norecord.js
shell.connect(__mysqluripwd);
var result = session.runSql('select 1');
session.close();

//@ ClassicResult help
result.help();

//@ ClassicResult help, \? [USE:ClassicResult help]
\? classicresult

//@ Help on affectedItemsCount
result.help('affectedItemsCount')

//@ Help on affectedItemsCount, \? [USE:Help on affectedItemsCount]
\? classicresult.affectedItemsCount

//@ Help on autoIncrementValue
result.help('autoIncrementValue')

//@ Help on autoIncrementValue, \? [USE:Help on autoIncrementValue]
\? classicresult.autoIncrementValue

//@ Help on columnCount
result.help('columnCount')

//@ Help on columnCount, \? [USE:Help on columnCount]
\? classicresult.columnCount

//@ Help on columnNames
result.help('columnNames')

//@ Help on columnNames, \? [USE:Help on columnNames]
\? classicresult.columnNames

//@ Help on columns
result.help('columns')

//@ Help on columns, \? [USE:Help on columns]
\? classicresult.columns

//@ Help on executionTime
result.help('executionTime')

//@ Help on executionTime, \? [USE:Help on executionTime]
\? classicresult.executionTime

//@ Help on info
result.help('info')

//@ Help on info, \? [USE:Help on info]
\? classicresult.info

//@ Help on warningsCount
result.help('warningsCount')

//@ Help on warningsCount, \? [USE:Help on warningsCount]
\? classicresult.warningsCount

//@ Help on warnings
result.help('warnings')

//@ Help on warnings, \? [USE:Help on warnings]
\? classicresult.warnings

//@ Help on fetchAll
result.help('fetchAll')

//@ Help on fetchAll, \? [USE:Help on fetchAll]
\? classicresult.fetchAll

//@ Help on fetchOne
result.help('fetchOne')

//@ Help on fetchOne, \? [USE:Help on fetchOne]
\? classicresult.fetchOne

//@ Help on fetchOneObject
result.help('fetchOneObject')

//@ Help on fetchOneObject, \? [USE:Help on fetchOneObject]
\? classicresult.fetchOneObject

//@ Help on getAffectedItemsCount
result.help('getAffectedItemsCount')

//@ Help on getAffectedItemsCount, \? [USE:Help on getAffectedItemsCount]
\? classicresult.getAffectedItemsCount

//@ Help on getAutoIncrementValue
result.help('getAutoIncrementValue')

//@ Help on getAutoIncrementValue, \? [USE:Help on getAutoIncrementValue]
\? classicresult.getAutoIncrementValue

//@ Help on getColumnCount
result.help('getColumnCount')

//@ Help on getColumnCount, \? [USE:Help on getColumnCount]
\? classicresult.getColumnCount

//@ Help on getColumnNames
result.help('getColumnNames')

//@ Help on getColumnNames, \? [USE:Help on getColumnNames]
\? classicresult.getColumnNames

//@ Help on getColumns
result.help('getColumns')

//@ Help on getColumns, \? [USE:Help on getColumns]
\? classicresult.getColumns

//@ Help on getExecutionTime
result.help('getExecutionTime')

//@ Help on getExecutionTime, \? [USE:Help on getExecutionTime]
\? classicresult.getExecutionTime

//@ Help on getInfo
result.help('getInfo')

//@ Help on getInfo, \? [USE:Help on getInfo]
\? classicresult.getInfo

//@ Help on getWarningsCount
result.help('getWarningsCount')

//@ Help on getWarningsCount, \? [USE:Help on getWarningsCount]
\? classicresult.getWarningsCount

//@ Help on getWarnings
result.help('getWarnings')

//@ Help on getWarnings, \? [USE:Help on getWarnings]
\? classicresult.getWarnings

//@ Help on hasData
result.help('hasData')

//@ Help on hasData, \? [USE:Help on hasData]
\? classicresult.hasData

//@ Help on help
result.help('help')

//@ Help on help, \? [USE:Help on help]
\? classicresult.help

//@ Help on nextResult
result.help('nextResult')

//@ Help on nextResult, \? [USE:Help on nextResult]
\? classicresult.nextResult

File: unittest/scripts/auto/py_shell/scripts/mysql_classic_result_help_norecord.py
shell.connect(__mysqluripwd)
classicresult = session.run_sql('select 1')
session.close()

#@ classicresult
classicresult.help()

#@ global ? for ClassicResult[USE:classicresult]
\? ClassicResult

#@ global help for ClassicResult[USE:classicresult]
\help ClassicResult

#@ classicresult.affected_items_count
classicresult.help('affected_items_count')

#@ global ? for affected_items_count[USE:classicresult.affected_items_count]
\? ClassicResult.affected_items_count

#@ global help for affected_items_count[USE:classicresult.affected_items_count]
\help ClassicResult.affected_items_count

#@ classicresult.auto_increment_value
classicresult.help('auto_increment_value')

#@ global ? for auto_increment_value[USE:classicresult.auto_increment_value]
\? ClassicResult.auto_increment_value

#@ global help for auto_increment_value[USE:classicresult.auto_increment_value]
\help ClassicResult.auto_increment_value

#@ classicresult.column_count
classicresult.help('column_count')

#@ global ? for column_count[USE:classicresult.column_count]
\? ClassicResult.column_count

#@ global help for column_count[USE:classicresult.column_count]
\help ClassicResult.column_count

#@ classicresult.column_names
classicresult.help('column_names')

#@ global ? for column_names[USE:classicresult.column_names]
\? ClassicResult.column_names

#@ global help for column_names[USE:classicresult.column_names]
\help ClassicResult.column_names

#@ classicresult.columns
classicresult.help('columns')

#@ global ? for columns[USE:classicresult.columns]
\? ClassicResult.columns

#@ global help for columns[USE:classicresult.columns]
\help ClassicResult.columns

#@ classicresult.execution_time
classicresult.help('execution_time')

#@ global ? for execution_time[USE:classicresult.execution_time]
\? ClassicResult.execution_time

#@ global help for execution_time[USE:classicresult.execution_time]
\help ClassicResult.execution_time

#@ classicresult.fetch_all
classicresult.help('fetch_all')

#@ global ? for fetch_all[USE:classicresult.fetch_all]
\? ClassicResult.fetch_all

#@ global help for fetch_all[USE:classicresult.fetch_all]
\help ClassicResult.fetch_all

#@ classicresult.fetch_one
classicresult.help('fetch_one')

#@ global ? for fetch_one[USE:classicresult.fetch_one]
\? ClassicResult.fetch_one

#@ global help for fetch_one[USE:classicresult.fetch_one]
\help ClassicResult.fetch_one

#@ classicresult.fetch_one_object
classicresult.help('fetch_one_object')

#@ global ? for fetch_one_object[USE:classicresult.fetch_one_object]
\? ClassicResult.fetch_one_object

#@ global help for fetch_one_object[USE:classicresult.fetch_one_object]
\help ClassicResult.fetch_one_object

#@ classicresult.get_affected_items_count
classicresult.help('get_affected_items_count')

#@ global ? for get_affected_items_count[USE:classicresult.get_affected_items_count]
\? ClassicResult.get_affected_items_count

#@ global help for get_affected_items_count[USE:classicresult.get_affected_items_count]
\help ClassicResult.get_affected_items_count

#@ classicresult.get_auto_increment_value
classicresult.help('get_auto_increment_value')

#@ global ? for get_auto_increment_value[USE:classicresult.get_auto_increment_value]
\? ClassicResult.get_auto_increment_value

#@ global help for get_auto_increment_value[USE:classicresult.get_auto_increment_value]
\help ClassicResult.get_auto_increment_value

#@ classicresult.get_column_count
classicresult.help('get_column_count')

#@ global ? for get_column_count[USE:classicresult.get_column_count]
\? ClassicResult.get_column_count

#@ global help for get_column_count[USE:classicresult.get_column_count]
\help ClassicResult.get_column_count

#@ classicresult.get_column_names
classicresult.help('get_column_names')

#@ global ? for get_column_names[USE:classicresult.get_column_names]
\? ClassicResult.get_column_names

#@ global help for get_column_names[USE:classicresult.get_column_names]
\help ClassicResult.get_column_names

#@ classicresult.get_columns
classicresult.help('get_columns')

#@ global ? for get_columns[USE:classicresult.get_columns]
\? ClassicResult.get_columns

#@ global help for get_columns[USE:classicresult.get_columns]
\help ClassicResult.get_columns

#@ classicresult.get_execution_time
classicresult.help('get_execution_time')

#@ global ? for get_execution_time[USE:classicresult.get_execution_time]
\? ClassicResult.get_execution_time

#@ global help for get_execution_time[USE:classicresult.get_execution_time]
\help ClassicResult.get_execution_time

#@ classicresult.get_info
classicresult.help('get_info')

#@ global ? for get_info[USE:classicresult.get_info]
\? ClassicResult.get_info

#@ global help for get_info[USE:classicresult.get_info]
\help ClassicResult.get_info

#@ classicresult.get_warnings_count
classicresult.help('get_warnings_count')

#@ global ? for get_warnings_count[USE:classicresult.get_warnings_count]
\? ClassicResult.get_warnings_count

#@ global help for get_warnings_count[USE:classicresult.get_warnings_count]
\help ClassicResult.get_warnings_count

#@ classicresult.get_warnings
classicresult.help('get_warnings')

#@ global ? for get_warnings[USE:classicresult.get_warnings]
\? ClassicResult.get_warnings

#@ global help for get_warnings[USE:classicresult.get_warnings]
\help ClassicResult.get_warnings

#@ classicresult.has_data
classicresult.help('has_data')

#@ global ? for has_data[USE:classicresult.has_data]
\? ClassicResult.has_data

#@ global help for has_data[USE:classicresult.has_data]
\help ClassicResult.has_data

#@ classicresult.help
classicresult.help('help')

#@ global ? for help[USE:classicresult.help]
\? ClassicResult.help

#@ global help for help[USE:classicresult.help]
\help ClassicResult.help

#@ classicresult.info
classicresult.help('info')

#@ global ? for info[USE:classicresult.info]
\? ClassicResult.info

#@ global help for info[USE:classicresult.info]
\help ClassicResult.info

#@ classicresult.next_result
classicresult.help('next_result')

#@ global ? for next_result[USE:classicresult.next_result]
\? ClassicResult.next_result

#@ global help for next_result[USE:classicresult.next_result]
\help ClassicResult.next_result

#@ classicresult.warnings_count
classicresult.help('warnings_count')

#@ global ? for warnings_count[USE:classicresult.warnings_count]
\? ClassicResult.warnings_count

#@ global help for warnings_count[USE:classicresult.warnings_count]
\help ClassicResult.warnings_count

#@ classicresult.warnings
classicresult.help('warnings')

#@ global ? for warnings[USE:classicresult.warnings]
\? ClassicResult.warnings

#@ global help for warnings[USE:classicresult.warnings]
\help ClassicResult.warnings
